05.ls/ls_5.rb

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+WHITE_SPACE = 2
+NUMBER_OF_COLUMNS = 3
+require 'etc'
+class Option
```

Comment:
`require`を先に書く方が自然かなと思います。
また、異なる種類の処理を書く前には改行を入れたいですね。

こんな感じがいいかなと思いますー

```suggestion
require 'etc'

WHITE_SPACE = 2
NUMBER_OF_COLUMNS = 3

class Option
```